crates/ioutil/src/lib.rs

```diff
@@ -1,3 +1,15 @@
 //! IO and formatting related utilies.
+
+use std::path::{Path, PathBuf};
 pub mod fmt;
 pub mod write;
+
+pub fn resolve_path(path: &Path) -> std::io::Result<PathBuf> {
+    if path.starts_with("~") {
```

Comment:
```suggestion
    if path.starts_with("~/") {
```

Maybe? Windows temp files are prefixed with tilde...

Reply:
I would hope that path.join on line 10 would avoid `//` on join (otherwise it would just be concat?) so I think we'd be safe to not do this.